2-mandatory/2-oxygen-levels.js

```diff
@@ -11,8 +11,28 @@
     Some string methods that might help you here are .replace() and .substring().
 */
 
-function findSafeOxygenLevel() {}
-
+function findSafeOxygenLevel(arrayOxygen) {
+  let tempNum=0;
+  let index=0;
+  let boo=true;
+  while (index <arrayOxygen.length && boo) {
```

Comment:
What other looping mechanism could you have used here that might be more appropriate?